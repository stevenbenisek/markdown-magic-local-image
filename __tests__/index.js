const path = require('path');
const transform = require('../index');

const config = {
  originalPath: path.resolve(__dirname),
};

test('LOCALIMAGE', () => {
  expect(transform(null, null, config)).toMatchSnapshot();
});

test('LOCALIMAGE:heading=false', () => {
  expect(transform(null, { heading: 'false' }, config)).toMatchSnapshot();
});

test('LOCALIMAGE:headingLevel=4', () => {
  expect(transform(null, { headingLevel: '4' }, config)).toMatchSnapshot();
});

test('LOCALIMAGE:include=**/*.svg', () => {
  expect(transform(null, { include: '**/*.svg' }, config)).toMatchSnapshot();
});
