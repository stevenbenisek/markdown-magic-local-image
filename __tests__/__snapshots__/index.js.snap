// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LOCALIMAGE 1`] = `
"###### test
![Alt text](__tests__/test.gif \\"test\\")
###### test
![Alt text](__tests__/test.jpg \\"test\\")
###### test
![Alt text](__tests__/test.png \\"test\\")
###### test
![Alt text](__tests__/test.svg \\"test\\")"
`;

exports[`LOCALIMAGE:heading=false 1`] = `
"![Alt text](__tests__/test.gif \\"test\\")
![Alt text](__tests__/test.jpg \\"test\\")
![Alt text](__tests__/test.png \\"test\\")
![Alt text](__tests__/test.svg \\"test\\")"
`;

exports[`LOCALIMAGE:headingLevel=4 1`] = `
"#### test
![Alt text](__tests__/test.gif \\"test\\")
#### test
![Alt text](__tests__/test.jpg \\"test\\")
#### test
![Alt text](__tests__/test.png \\"test\\")
#### test
![Alt text](__tests__/test.svg \\"test\\")"
`;

exports[`LOCALIMAGE:include=**/*.svg 1`] = `
"###### test
![Alt text](__tests__/test.svg \\"test\\")"
`;
